sycl/doc/extensions/GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc

```diff
@@ -246,8 +325,8 @@ the default sorting algorithm defined by the sorter calling by `joint_sort`.
 `range_size` represents a range size for sorting,
 e.g. `last-first` from `operator()` arguments.
 Result depends on the `scope` parameter:
-use `sycl::memory_scope::work_group` to get memory size required for each work_group;
-use `sycl::memory_scope::sub_group` to get memory size required for each sub_group`.
+use `sycl::memory_scope::work_group` to get memory size required for each work-group;
+use `sycl::memory_scope::sub_group` to get memory size required for each sub-group`.

```

Comment:
Is there an extra backtick at the end of this line?

Reply:
Yes... Thanks. It would be great if CI checks could catch that